[2019-12-30 16:37:44,236] [log.py|__console] [line:40] INFO    : 当前执行测试用例ID-> test_login001.3 ; 测试点-> 手机号为空登录
[2019-12-30 16:37:44,597] [log.py|__console] [line:40] INFO    : 检查点-> 用户名不能为空！
[2019-12-30 16:37:44,605] [log.py|__console] [line:40] INFO    : 异常登录，返回实际结果是->: 用户名不能为空！
[2019-12-30 16:37:44,956] [log.py|__console] [line:40] INFO    : 当前执行测试用例ID-> test_login001.4 ; 测试点-> 密码为空登录
[2019-12-30 16:37:45,347] [log.py|__console] [line:40] INFO    : 检查点-> 密码不能为空！
[2019-12-30 16:37:45,353] [log.py|__console] [line:40] INFO    : 异常登录，返回实际结果是->: 密码不能为空！
